ui/app/(wrapped)/ressources/page.tsx

```diff
@@ -21,7 +19,6 @@ export default async function Ressources() {
   return (
     <>
```

Comment:
Tu peux supprimer ça du coup

```suggestion
    return (<Doc recordMap={recordMap}>)
```